torchbox/torchbox.com
# TWE-1 - BE - New homepage hero fields

[Link to Ticket](https://torchbox.atlassian.net/browse/TWE-1)

### Description of Changes Made

This PR replaces the current introduction text field with a rich text field, and adds 2 new fields for the heading shown in the hero area of the homepage.

Styling will be done in a separate ticket.

### How to Test

1. Apply the migrations.
2. Edit the homepage.
3. You should see the new fields.
4. Editing the new fields should display the new content on the preview.

### Screenshots

<details><summary>Editor</summary>

![image](https://github.com/user-attachments/assets/dc1d0a16-d7f9-41f3-84ef-59121afdf7dd)

</details>

### MR Checklist


#### Unit tests


#### Documentation


#### Browser testing

  - Latest version of Chrome on mac
  - Latest version of Firefox on mac
  - Latest version of Safari on mac
  - Safari on last two versions of iOS
  - Chrome on last two versions of Android

#### Data protection


#### Light and dark mode


#### Accessibility


#### Sustainability


#### Pattern library
